carp.common/src/commonMain/kotlin/dk/cachet/carp/common/EmailAddress.kt

```diff
@@ -1,11 +1,22 @@
 package dk.cachet.carp.common
 
+import dk.cachet.carp.common.serialization.createCarpStringPrimitiveSerializer
+import kotlinx.serialization.KSerializer
 import kotlinx.serialization.Serializable
 
 
 /**
  * Represents the address of an electronic mail sender or recipient.
  * TODO: Validate so only valid email addresses can be passed.
  */
-@Serializable
+@Serializable( EmailAddressSerializer::class )
 data class EmailAddress( val address: String )
```

Comment:
Email should have some kind of input validation. An email regex for instance

Reply:
The TODO is there. Out of scope for this PR.